lib/protocol/network.js

```diff
@@ -303,7 +302,7 @@ class Network {
 
   /**
    * Get a network by its base58 address prefix.
-   * @param {Number} value
+   * @param {Number} prefix
```

Comment:
this is actually a `String` example `'xpub'` for mainnet

Reply:
I think it's a `Number` check `cmpBase58()`